src/java.naming/share/classes/module-info.java

```diff
@@ -36,21 +36,33 @@
  * The following implementation specific environment properties are supported by the
  * default LDAP Naming Service Provider implementation in the JDK:
  * <ul>
+ *     <li>{@code java.naming.ldap.factory.socket}:
+ *         <br>The value of this environment property specifies the fully
+ *         qualified class name of the socket factory used by the LDAP provider.
+ *         This class must implement the javax.net.SocketFactory abstract class
```

Comment:
You could add a link to the `SocketFactory` class here: 
```suggestion
 *         This class must implement the {@link javax.net.SocketFactory} abstract class
```

Reply:
Done.